integration-tests/utils/terra-envs.js

```diff
@@ -0,0 +1,27 @@
+module.exports = {
+  local: {
+    billingProject: 'saturn-integration-test-dev',
+    testUrl: 'http://localhost:3000',
+    workflowName: 'echo_to_file'
+  },
+  dev: {
+    billingProject: 'saturn-integration-test-dev',
+    testUrl: 'https://bvdp-saturn-dev.appspot.com',
+    workflowName: 'echo_to_file'
```

Comment:
TOL: Going forward we should think about which things should typically be allowed to vary between environments, and which things are expected to be standard.

Reply:
I agree when it comes to `workflowName`. I don't remember exactly why I put it here but not other things like `userEmail`. In both cases (and possibly others), they are things that need to be set up intentionally in each environment, however in practice the values are the same in all environments.